main.go

```diff
@@ -43,53 +37,53 @@ func init() {
 		"Token URL.",
 	)
 	flag.StringVar(
-		&args.gatewayURL,
-		"gateway-url",
-		"http://localhost:8000",
-		"Gateway URL.",
-	)
-	flag.StringVar(
-		&args.clientID,
-		"client-id",
-		"cloud-services",
-		"OpenID client identifier.",
-	)
-	flag.StringVar(
-		&args.clientSecret,
-		"client-secret",
-		"",
-		"OpenID client secret.",
-	)
-	flag.StringVar(
-		&args.token,
-		"token",
-		"",
-		"Offline token for authentication.",
-	)
-	flag.IntVar(
-		&args.durationInMin,
-		"duration-in-min",
-		1,
-		"How long should the load test take.",
-	)
-	flag.StringVar(
-		&args.outputDirectory,
-		"output-path",
+		&args.configFile,
+		"config-file",
 		"",
-		"path to output results.",
+		"Test config file",
 	)
 }
 
 func main() {
 	flag.Parse()
 
-	// Consider passing a yaml map from test-name to rate as config file to allow more flexability.
+	// set defaults
+	duration := time.Duration(1) * time.Minute
+	outputDirectory := "./results"
```

Comment:
Just to shed light on this - this change fixes the "results output directory" to `./results` - am I reading this correct?

cc: @chentex @kurtismullins

Reply:
Oh god, that is a fail on my side, great catch, I will fix that now